server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesOutboxFlagUpdateTest.java

```diff
@@ -94,14 +95,16 @@ public void enQueue(Mail mail) {
                 }
 
                 @Override
-                public MailQueueItem deQueue() {
-                    CountDownLatch blockingLatch = new CountDownLatch(1);
-                    try {
-                        blockingLatch.await();
-                    } catch (InterruptedException e) {
-                        throw new RuntimeException(e);
-                    }
-                    return null;
+                public Flux<MailQueueItem> deQueue() {
```

Comment:
Let me guess: What you wrotte is just an overly complexe `Flux.never()`?

Reply:
You are probably right, I didn't analyze what the previous code was actually doing, but it seems that `Flux.never` is more than enough.